src/qt/bitcoin.cpp

```diff
@@ -390,7 +390,7 @@ void BitcoinApplication::shutdownResult()
 
 void BitcoinApplication::handleRunawayException(const QString &message)
 {
-    QMessageBox::critical(nullptr, "Runaway exception", BitcoinGUI::tr("A fatal error occurred. Bitcoin can no longer continue safely and will quit.") + QString("\n\n") + message);
+    QMessageBox::critical(nullptr, "Runaway exception", BitcoinGUI::tr("A fatal error occurred. Elements can no longer continue safely and will quit.") + QString("\n\n") + message);
```

Comment:
This is a key that is used to index into the translation. If we change the string here without changing the <source> elements in the locale specific translations, the localized builds may fail to display a string.
For example:

https://github.com/ElementsProject/elements/blob/master/src/qt/locale/bitcoin_es.ts#L576

Reply:
Yeah it seemed that that's what we have been doing in the past.